Mathlib/Tactic/UnitInterval.lean

```diff
@@ -0,0 +1,6 @@
+/-- A tactic that solves `0 ≤ ↑x`, `0 ≤ 1 - ↑x`, `↑x ≤ 1`, and `1 - ↑x ≤ 1` for `x : I`. -/
```

Comment:
```suggestion
/-- A tactic that solves `0 ≤ x`, `0 ≤ 1 - x`, `x ≤ 1`, and `1 - x ≤ 1` for `x : I`. -/
```
I think these arrows won't appear in the pretty printed expression in Lean 4.

Reply:
By default, they do atm.  But there were some requests to hide them (and we will add an option for that, not sure about the default though).